test/Chainweb/Test/Golden.hs

```diff
@@ -0,0 +1,40 @@
+-- |
+-- Module: Chainweb.Test.Golden
+-- Copyright: Copyright © 2019 Kadena LLC.
+-- License: See LICENSE file
+-- Maintainer: Colin Woodbury <colin@kadena.io>
+-- Stability: experimental
+--
+-- Utilities for working with `tasty-golden`.
+
+module Chainweb.Test.Golden where
+
+import qualified Data.ByteString.Lazy as BL
+
+import Test.Tasty (TestTree)
+import Test.Tasty.Golden (goldenVsString)
+
+-- internal modules
+
+import Chainweb.Test.Utils (ScheduledTest(..))
+
+---
+
+goldenFilesDir :: FilePath
+goldenFilesDir = "test/golden/"
+
+golden
```

Comment:
@larskuhtz should these just be in `Chainweb.Test.Utils` instead?

Reply:
Maybe rename the module to `Chainweb.Test.Utils.Golden` to make it clear that it's test utils. Currently, it's sometimes a bit difficult to find utils because there is no distinction between utility and test module namespaces.

Reply:
Actually, I think an internal library for test utils would make sense, but let's not discuss this here.

Reply:
I'll do a quick renaming.